### ISSUE TITLE
Heading: align breakpoints with AXA UI Kit

### ISSUE THREAD

**Issue by MarekLacoAXA** [axa-ch/patterns-library#1739]
Heading has three breakpoints for font sizes, see:

```css
h1.a-heading {
  @include typo-title(large-1);

  @include breakpoint($mediaquery-md) {
    @include typo-title(huge-1);
  }

  @include breakpoint($mediaquery-lg-up) {
    @include typo-title(huge-3);
  }
```

But AXA UI Kit has only 2 breakpoints:

https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=0%3A58

![image](https://user-images.githubusercontent.com/43809369/79950852-faaf7980-8477-11ea-94f4-680b962af37e.png)

This should be aligned: two breakpoints in UI Kit meas Heading should have two breakpoints also.
(as discussed with Jonas on 21.4.2020).

**Comment by raphaellueckl** [comment]
This is due to the fact, that typography wasn't really finished by the time we started implementing the heading component (there were some open questions), so we decided to just take our usual title typography we already had. There are also very minor differences of other details, compared to the figma version.

The current heading styles were basically a copy pasta of the material typo. When the typography is finished, we can have a look at it again and maybe change the materials / axa-text / axa-heading at once.
